IHP/IDE/ToolServer/ViewContext.hs

```diff
@@ -37,3 +41,12 @@ findWebControllers = do
     directoryFiles <-  listDirectory "Web/Controller"
     let controllerFiles :: [Text] =  filter (\x -> not $ "Prelude" `isInfixOf` x || "Context" `isInfixOf` x)  $ map cs directoryFiles
     pure $ map (Text.replace ".hs" "") controllerFiles
+
+
+findApps :: IO ([Text])
+findApps = do
+    mainhs <- IO.readFile "Main.hs"
```

Comment:
[Can we `readFile` from `Data.Text.IO` here?](https://hackage.haskell.org/package/text-1.2.4.0/docs/Data-Text-IO.html#v:readFile) That returns a Text and is a lot more efficient